### PR TITLE
NAIP and Discover blog post

### PR DESCRIPTION
**Pull request by rkelson**
this PR needs to be reviewed and okay'd before being merged.
I will add a comment when the PR is okay'd and ready to merge.

The merge needs to happen early morning of 2/23/17 after Calvin makes some changes to Discover at 8 am eastern time

### CODE REVIEW

**Comment by steveoh** [comment]
@stdavis will you please merge this 10 minutes before 8 tomorrow! Thanks.

File: _posts/2017-02-21-new-services-on-discover.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+Recently AGRC made some changes to the services available from our cloud server Discover (discover.agrc.utah.gov).
+To see these new enhancements you may need to _refresh_ the connection you use to access Discover. ArcMap users can
+right-click your connection to Discover in ArcCatalog and select `Refresh`. You might also want to clear your local cache.
+ArcMap users can go to the service’s `Layer Properties -> Cache` tab and select `Clear Local Cache Now`. Be patient
+as this could take several minutes. Another option is to completely clear your ArcMap cache by going
+to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.  
+
+The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
```
Comment (steveoh) [inline _posts/2017-02-21-new-services-on-discover.md]: > layer's display title only a does not have any effect on the urls or ...

should say `only and does`

File: _posts/2017-02-21-new-services-on-discover.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+ - **Utah** (licensed) - This service is now a mix of the most recent NAIP 1-meter and Google 6-inch imagery. The service
+ now utilizes scale dependent rendering to display the color balanced NAIP imagery down to cache level 16,
+ then renders the Google imagery at level 17 (starting at approx. 1:6,500) and down to level 20.
+
+ - **Google** (licensed)- This new service is the replacement for the old **utah** service and has the
```
Comment (steveoh) [inline _posts/2017-02-21-new-services-on-discover.md]: Your above text has a space before the `-` and the others don't. would you update those.

File: _posts/2017-02-21-new-services-on-discover.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ most recent Google imagery for Utah.
+
+ - **Basemap-Hybrid** (licensed)- This new service was created for convenience and performance for a dynamic mash-up of vector features with a transparent background from the **Basemap-Overlay** and the **Utah** imagery service.
+
+More information about the services from Discover can be found [here]({{"/data/sgid-base-map-services-arcmap/" | prepend:site.baseurl}}).
```
Comment (steveoh) [inline _posts/2017-02-21-new-services-on-discover.md]: `here` links are kind of rookie. Could you update that to something like 

More information about the services from Discover can be found in the [Using SGID Base Map & Imagery Services in ArcMap](#)

**Comment by rkelson** [comment]
thanks for the second set of eyes

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by rkelson** [comment]
this PR is ready to merge

**Pull request merged**